maestro-orchestra-models/src/main/java/maestro/orchestra/MaestroCommand.kt

```diff
@@ -19,88 +19,50 @@
+ * serialization logic. The earlier implementation of this class had a nullable field for
+ * each command. Sometime in the future we may move this serialization logic to the backend
+ * itself, where it would be more relevant.
+ */
+@JsonSerialize(using = MaestroCommandSerializer::class)
 data class MaestroCommand(
-    val tapOnElement: TapOnElementCommand? = null,
-    val tapOnPoint: TapOnPointCommand? = null,
-    val scrollCommand: ScrollCommand? = null,
-    val swipeCommand: SwipeCommand? = null,
-    val backPressCommand: BackPressCommand? = null,
-    val assertCommand: AssertCommand? = null,
-    val inputTextCommand: InputTextCommand? = null,
-    val launchAppCommand: LaunchAppCommand? = null,
-    val applyConfigurationCommand: ApplyConfigurationCommand? = null,
-    val openLinkCommand: OpenLinkCommand? = null,
-    val pressKeyCommand: PressKeyCommand? = null,
-    val eraseTextCommand: EraseTextCommand? = null,
+    val command: Command?,
 ) {
 
     fun injectEnv(envParameters: Map<String, String>): MaestroCommand {
-        return copy(
-            tapOnElement = tapOnElement?.injectEnv(envParameters),
-            tapOnPoint = tapOnPoint?.injectEnv(envParameters),
-            scrollCommand = scrollCommand?.injectEnv(envParameters),
-            swipeCommand = swipeCommand?.injectEnv(envParameters),
-            backPressCommand = backPressCommand?.injectEnv(envParameters),
-            assertCommand = assertCommand?.injectEnv(envParameters),
-            inputTextCommand = inputTextCommand?.injectEnv(envParameters),
-            launchAppCommand = launchAppCommand?.injectEnv(envParameters),
-            applyConfigurationCommand = applyConfigurationCommand?.injectEnv(envParameters),
-            openLinkCommand = openLinkCommand?.injectEnv(envParameters),
-            pressKeyCommand = pressKeyCommand?.injectEnv(envParameters),
-            eraseTextCommand = eraseTextCommand?.injectEnv(envParameters),
-        )
+        return copy(command = command?.injectEnv(envParameters))
     }
 
     fun description(): String {
-        tapOnElement?.let {
-            return it.description()
-        }
-
-        tapOnPoint?.let {
-            return it.description()
-        }
-
-        scrollCommand?.let {
-            return it.description()
-        }
-
-        swipeCommand?.let {
-            return it.description()
-        }
-
-        backPressCommand?.let {
-            return it.description()
-        }
-
-        assertCommand?.let {
-            return it.description()
-        }
-
-        inputTextCommand?.let {
-            return it.description()
-        }
-
-        launchAppCommand?.let {
-            return it.description()
-        }
-
-        applyConfigurationCommand?.let {
-            return it.description()
-        }
-
-        openLinkCommand?.let {
-            return it.description()
-        }
+        return command?.description() ?: "No op"
```

Comment:
The only reason `command` is nullable here is because of this function returning a `"No op"` value. Is this value used in the backend? If not, we could make `command` a non-null property member in `MaestroCommand` and also get rid of the associated test from `MaestroCommandTest`.